File: code_llm/zero-shot/gemini-3.1-pro-preview/Relógio.cpp
#include <iostream>

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    long long H, M, S, T;
    if (cin >> H >> M >> S >> T) {
        long long total_seconds = H * 3600 + M * 60 + S;
        total_seconds += T;
        
        total_seconds %= 86400;
        
        long long new_H = total_seconds / 3600;
        total_seconds %= 3600;
        
        long long new_M = total_seconds / 60;
        long long new_S = total_seconds % 60;
        
        cout << new_H << "\n" << new_M << "\n" << new_S << "\n";
    }
    
    return 0;
}

File: output/z_ai_glm_5/few/python_noimg/Relógio.py
import sys

input_data = sys.stdin.read().split()
if not input_data:
    exit()

H = int(input_data[0])
M = int(input_data[1])
S = int(input_data[2])
T = int(input_data[3])

total_seconds = H * 3600 + M * 60 + S + T
total_seconds %= 86400

new_H = total_seconds // 3600
new_M = (total_seconds % 3600) // 60
new_S = total_seconds % 60

print(new_H)
print(new_M)
print(new_S)